SQL/0000-00-04-Help.sql

```diff
@@ -22,7 +22,7 @@ CREATE TABLE `help` (
or page.\r\n<b>Dashboard</b>\r\nUpon logging into LORIS, users will see the Dashboard.\r\n<b>Menu Bar</b>\rThe top Menu Bar provides easy access to LORIS’ many modules and features. To return to the main Dashboard at any time, click on “LORIS” at the far left corner of the Menu Bar. Key user information is displayed at the right side of the Menu Bar, including the User’s name and the sites to which the user belongs. To log out from LORIS, click on the User’s name and select Log Out from the drop-down menu, or select User Preferences to update user information, profile settings, and to change your password.\r\nClick on any of the six main menus to access LORIS modules:\r\n(Note not all modules may be visible on Demo):\r<b>Candidate:</b> New Profile, Access Profile\r<b>Clinical:</b> Reliability, Conflict Resolver, Examiner, Training, Media\r<b>Imaging:</b> Radiological Reviews, DICOM Archive, Imaging Browser, MRI Violated Scans, Imaging Uploader\r<b>Reports:</b> Statistics, Data Query Tool\r<b>Tools:</b> Data Dictionary, Document Repository, Data Integrity Flag, Data Team Helper, Instrument Builder, Genomic Browser, Data Release, Acknowledgments, Issue Tracker\r<b>Admin:</b> User Accounts, Survey Module, Help Editor, Instrument Manager, Configuration, Server Processes Manager\rIn smaller browser windows or on mobile devices (e.g. tablet), these menus may be hidden and can be accessed instead by clicking the downward arrow in the Menu Bar.\r\n<b>Dashboard Features</b>\rThe Dashboard is composed of panels showing at-a-glance status: Recruitment, Study Progression, My Tasks, Document Repository Notifications, and Behavioral Feedback Notifications. Each panel can be hidden by clicking the arrow located in the top right corner.\r\n<b>Recruitment:</b> View how many candidates have been recruited for a given study.  Click the “Views” button in the right-hand corner for a breakdown by site. Note that charts include data from ineligible, excluded, and consent-withdrawn candidates.\r\n<b>Study Progression:</b> View a timeline of scans collected by each study site, or a timeline of candidate recruitment per site. Zoom in on the chart to view detailed data points. Note that charts include data from ineligible, excluded, and consent withdrawn candidates.\r\n<b>My Tasks:</b> User-specific list of outstanding tasks such as data entry conflicts, incomplete forms, or new and pending scans awaiting review. Click on any task panel to jump to the appropriate module.\r\n<b>Document Repository Notifications:</b> View recently uploaded documents for your study or site user’s study or site.  Click on a document name to download the file directly from the Dashboard. Click on “Document Repository” to view the complete list of available documents.\r\n<b>Behavioral Feedback Notifications:<\b> Track issues and corrections to behavioural/clinical data points. Click on a notification to redirect to the instrument with new changes or feedback. \r\n','2014-09-01 00:00:00',NULL),
```

Comment:
`<b>Behavioral Feedback Notifications:<\b>`
This tag doesn't close properly and the entire line is bold

Reply:
There should be better paragraph spacing as well